[BUG] Publishing tvOS application with SkiaSharp fails

### Description

When I am trying to publish a tvOS app with SkiaSharp referenced, compilation fails.
It works well when I am running in debug with simulator though. 

### Code

Pretty much any usage of SkiaSharp. For example:

```cs
using SkiaSharp;
using tv_test;

var info = new SKImageInfo(512, 512);
var surface = SKSurface.Create(info);
var canvas = surface.Canvas;
canvas.Clear(SKColors.White);

UIApplication.Main (args, null, typeof (AppDelegate));

```

To run:
```
dotnet publish -c Release -r tvos-arm64
```

[Archive.zip](https://github.com/mono/SkiaSharp/files/13930830/Archive.zip)

### Expected Behavior

Application can be compiled.

### Actual Behavior

Compilation fails

### Version of SkiaSharp

2.88.3 (Current)

### Last Known Good Version of SkiaSharp

Other (Please indicate in the description)

### IDE / Editor

Other (Please indicate in the description)

### Platform / Operating System

tvOS

### Platform / Operating System Version

tvOS: 13.0+

### Devices

Apple TV

### Relevant Screenshots

_No response_

### Relevant Log Output

```shell
tvtest -> /Users/maxkatz6/repos/trash-bin/tvtest/bin/Release/net8.0-tvos/tvos-arm64/tvtest.dll
  Optimizing assemblies for size may change the behavior of the app. Be sure to test after publishing. See: https://aka.ms/dotnet-illink
  Optimizing assemblies for size. This process might take a while.
  Tool xcrun execution finished (exit code = 1).
          
  Undefined symbols for architecture arm64:
    "_DeleteCriticalSection", referenced from:
        wrapper_managed_to_native_SkiaSharp_Internals_PlatformLock_NonAlertableWin32Lock_DeleteCriticalSection_intptr in SkiaSharp.dll.o
       (maybe you meant: _p_112_plt_SkiaSharp_SkiaSharp_Internals_PlatformLock_NonAlertableWin32Lock_DeleteCriticalSection_intptr_llvm)
    "_EnterCriticalSection", referenced from:
        wrapper_managed_to_native_SkiaSharp_Internals_PlatformLock_NonAlertableWin32Lock_EnterCriticalSection_intptr in SkiaSharp.dll.o
       (maybe you meant: _p_114_plt_SkiaSharp_SkiaSharp_Internals_PlatformLock_NonAlertableWin32Lock_EnterCriticalSection_intptr_llvm)
    "_InitializeCriticalSectionEx", referenced from:
        wrapper_managed_to_native_SkiaSharp_Internals_PlatformLock_NonAlertableWin32Lock_InitializeCriticalSectionEx_intptr_uint_uint in SkiaSharp.dll.o
       (maybe you meant: _p_111_plt_SkiaSharp_SkiaSharp_Internals_PlatformLock_NonAlertableWin32Lock_InitializeCriticalSectionEx_intptr_uint_uint_llvm)
    "_LeaveCriticalSection", referenced from:
        wrapper_managed_to_native_SkiaSharp_Internals_PlatformLock_NonAlertableWin32Lock_LeaveCriticalSection_intptr in SkiaSharp.dll.o
       (maybe you meant: _p_115_plt_SkiaSharp_SkiaSharp_Internals_PlatformLock_NonAlertableWin32Lock_LeaveCriticalSection_intptr_llvm)
    "_gnu_get_libc_version", referenced from:
        wrapper_managed_to_native_SkiaSharp_Internals_PlatformConfiguration_gnu_get_libc_version in SkiaSharp.dll.o
       (maybe you meant: _p_145_plt_SkiaSharp_SkiaSharp_Internals_PlatformConfiguration_gnu_get_libc_version_llvm)
  ld: symbol(s) not found for architecture arm64
  clang: error: linker command failed with exit code 1 (use -v to see invocation)
  
/usr/local/share/dotnet/packs/Microsoft.tvOS.Sdk/17.0.8490/targets/Xamarin.Shared.Sdk.targets(1556,3): error : clang++ exited with code 1: [/Users/maxkatz6/repos/trash-bin/tvtest/tvtest.csproj]
/usr/local/share/dotnet/packs/Microsoft.tvOS.Sdk/17.0.8490/targets/Xamarin.Shared.Sdk.targets(1556,3): error : Undefined symbols for architecture arm64: [/Users/maxkatz6/repos/trash-bin/tvtest/tvtest.csproj]
/usr/local/share/dotnet/packs/Microsoft.tvOS.Sdk/17.0.8490/targets/Xamarin.Shared.Sdk.targets(1556,3): error :   "_DeleteCriticalSection", referenced from: [/Users/maxkatz6/repos/trash-bin/tvtest/tvtest.csproj]
/usr/local/share/dotnet/packs/Microsoft.tvOS.Sdk/17.0.8490/targets/Xamarin.Shared.Sdk.targets(1556,3): error :       wrapper_managed_to_native_SkiaSharp_Internals_PlatformLock_NonAlertableWin32Lock_DeleteCriticalSection_intptr in SkiaSharp.dll.o [/Users/maxkatz6/repos/trash-bin/tvtest/tvtest.csproj]
/usr/local/share/dotnet/packs/Microsoft.tvOS.Sdk/17.0.8490/targets/Xamarin.Shared.Sdk.targets(1556,3): error :      (maybe you meant: _p_112_plt_SkiaSharp_SkiaSharp_Internals_PlatformLock_NonAlertableWin32Lock_DeleteCriticalSection_intptr_llvm) [/Users/maxkatz6/repos/trash-bin/tvtest/tvtest.csproj]
/usr/local/share/dotnet/packs/Microsoft.tvOS.Sdk/17.0.8490/targets/Xamarin.Shared.Sdk.targets(1556,3): error :   "_EnterCriticalSection", referenced from: [/Users/maxkatz6/repos/trash-bin/tvtest/tvtest.csproj]
/usr/local/share/dotnet/packs/Microsoft.tvOS.Sdk/17.0.8490/targets/Xamarin.Shared.Sdk.targets(1556,3): error :       wrapper_managed_to_native_SkiaSharp_Internals_PlatformLock_NonAlertableWin32Lock_EnterCriticalSection_intptr in SkiaSharp.dll.o [/Users/maxkatz6/repos/trash-bin/tvtest/tvtest.csproj]
/usr/local/share/dotnet/packs/Microsoft.tvOS.Sdk/17.0.8490/targets/Xamarin.Shared.Sdk.targets(1556,3): error :      (maybe you meant: _p_114_plt_SkiaSharp_SkiaSharp_Internals_PlatformLock_NonAlertableWin32Lock_EnterCriticalSection_intptr_llvm) [/Users/maxkatz6/repos/trash-bin/tvtest/tvtest.csproj]
/usr/local/share/dotnet/packs/Microsoft.tvOS.Sdk/17.0.8490/targets/Xamarin.Shared.Sdk.targets(1556,3): error :   "_InitializeCriticalSectionEx", referenced from: [/Users/maxkatz6/repos/trash-bin/tvtest/tvtest.csproj]
/usr/local/share/dotnet/packs/Microsoft.tvOS.Sdk/17.0.8490/targets/Xamarin.Shared.Sdk.targets(1556,3): error :       wrapper_managed_to_native_SkiaSharp_Internals_PlatformLock_NonAlertableWin32Lock_InitializeCriticalSectionEx_intptr_uint_uint in SkiaSharp.dll.o [/Users/maxkatz6/repos/trash-bin/tvtest/tvtest.csproj]
/usr/local/share/dotnet/packs/Microsoft.tvOS.Sdk/17.0.8490/targets/Xamarin.Shared.Sdk.targets(1556,3): error :      (maybe you meant: _p_111_plt_SkiaSharp_SkiaSharp_Internals_PlatformLock_NonAlertableWin32Lock_InitializeCriticalSectionEx_intptr_uint_uint_llvm) [/Users/maxkatz6/repos/trash-bin/tvtest/tvtest.csproj]
/usr/local/share/dotnet/packs/Microsoft.tvOS.Sdk/17.0.8490/targets/Xamarin.Shared.Sdk.targets(1556,3): error :   "_L [/Users/maxkatz6/repos/trash-bin/tvtest/tvtest.csproj]
```


### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/binding/Binding.Shared/PlatformConfiguration.cs
+++ b/binding/Binding.Shared/PlatformConfiguration.cs
@@ -18,41 +18,51 @@ namespace SkiaSharp.Internals
 	{
 		private const string LibCLibrary = "libc";
 
-		public static bool IsUnix { get; }
+		public static bool IsUnix => IsMac || IsLinux;
 
-		public static bool IsWindows { get; }
-
-		public static bool IsMac { get; }
-
-		public static bool IsLinux { get; }
-
-		public static bool IsArm { get; }
-
-		public static bool Is64Bit { get; }
-
-		static PlatformConfiguration ()
-		{
+		public static bool IsWindows {
 #if WINDOWS_UWP
-			IsMac = false;
-			IsLinux = false;
-			IsUnix = false;
-			IsWindows = true;
-
-			var arch = Package.Current.Id.Architecture;
-			const ProcessorArchitecture arm64 = (ProcessorArchitecture)12;
-			IsArm = arch == ProcessorArchitecture.Arm || arch == arm64;
+			get => true;
+#elif NET6_0_OR_GREATER
+			get => OperatingSystem.IsWindows ();
 #else
-			IsMac = RuntimeInformation.IsOSPlatform (OSPlatform.OSX);
-			IsLinux = RuntimeInformation.IsOSPlatform (OSPlatform.Linux);
-			IsUnix = IsMac || IsLinux;
-			IsWindows = RuntimeInformation.IsOSPlatform (OSPlatform.Windows);
-
-			var arch = RuntimeInformation.ProcessArchitecture;
-			IsArm = arch == Architecture.Arm || arch == Architecture.Arm64;
+			get => RuntimeInformation.IsOSPlatform (OSPlatform.Windows);
 #endif
-
-			Is64Bit = IntPtr.Size == 8;
 		}
+
+		public static bool IsMac {
+#if WINDOWS_UWP
+			get => false;
+#elif NET6_0_OR_GREATER
+			get => OperatingSystem.IsMacOS ();
+#else
+			get => RuntimeInformation.IsOSPlatform (OSPlatform.OSX);
+#endif
+		}
+
+		public static bool IsLinux {
+#if WINDOWS_UWP
+			get => false;
+#elif NET6_0_OR_GREATER
+			get => OperatingSystem.IsLinux ();
+#else
+			get => RuntimeInformation.IsOSPlatform (OSPlatform.Linux);
+#endif
+		}
+
+		public static bool IsArm {
+#if WINDOWS_UWP
+			get {
+				var arch = Package.Current.Id.Architecture;
+				const ProcessorArchitecture arm64 = (ProcessorArchitecture)12;
+				return arch == ProcessorArchitecture.Arm || arch == arm64;
+			}
+#else
+			get => RuntimeInformation.ProcessArchitecture is Architecture.Arm or Architecture.Arm64;
+#endif
+		}
+
+		public static bool Is64Bit => IntPtr.Size == 8;
 
 		private static string linuxFlavor;
 

